secprofile/oci_profile.go

```diff
@@ -20,12 +36,17 @@ var OCIProfileType = ProfileType("oci-profile")
 type OCIProfile struct {
 	OCI           *specs.Spec
 	AppArmorSetup *apparmor.ProfileData
+	APIAccess     *APIAccessConfig
 }
 
 // NewOCIProfile instantiates an OCIProfile object with an OCI specification structure
 func NewOCIProfile(ociSpec *specs.Spec, apparmorProfileName string) *OCIProfile {
 	if apparmorProfileName == "" || apparmorProfileName == "unconfined" {
-		return &OCIProfile{OCI: ociSpec, AppArmorSetup: nil}
+		return &OCIProfile{
+			OCI:           ociSpec,
+			AppArmorSetup: nil,
+			APIAccess:     &APIAccessConfig{make(map[APIID]map[APISubsetId]APIAccess)},
```

Comment:
I think the parsed APIAccess string needs to be passed to `NewOCIProfile` function